### DISCUSSION TITLE
What is requires to purchase?

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#123] [Ideas]
On this screen : 
![Screenshot 2023-06-07 at 22 19 11](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/e476d396-f221-4ec4-a0f9-d8b02fffbe4c)
I am proposing here to incorporate small icons that provide direct indications of the specific requirements for making a purchase. In certain cases, users may want to know whether a purchase necessitates a credit card transaction and passing Know Your Customer (KYC) verification, or if it simply requires having USDC (USD Coin) in their wallet. By offering these details about the purchase process, we can prevent users from mistakenly selecting the default option (the first on the list) that may not meet their requirements. This will save them from spending more than a minute exploring unnecessary choices and potentially abandoning the purchase altogether, resulting in lost token sales.

**Comment by TJ-BitCanna** [comment]
Good idea to lower the amount of surprises :)
